kotest-property/src/jvmMain/kotlin/io/kotest/property/arbitrary/bigdecimal.kt

```diff
@@ -28,8 +28,11 @@ fun Arb.Companion.bigDecimal(scale: Int, roundingMode: RoundingMode) =
    bigDecimal().map { it.setScale(scale, roundingMode) }
 
 fun Arb.Companion.bigDecimal(min: BigDecimal, max: BigDecimal): Arb<BigDecimal> {
-   return arbitrary(bigDecimalEdgecases) {
+   val boundedEdgecases = bigDecimalDefaultEdgecases
+      .filter { min <= it && it < max }
+      .plus(min)
+
+   return arbitrary(boundedEdgecases) {
       min.add(BigDecimal(Math.random()).multiply(max.subtract(min)))
```

Comment:
@ciffer thanks heaps! do you mind also fixing this bug? It seems like currently this arb ignores the provided random source
```suggestion
      min.add(BigDecimal(it.random.nextDouble()).multiply(max.subtract(min)))
```

Reply:
I'll do that later today :+1: